src/data/_contributors.yaml

```diff
@@ -1546,3 +1546,20 @@ maciejzasada:
     - author
   google: "104812516714503133849"
   email: maciej@unit9.com
+
+_satyakresna:
```

Comment:
name keys cannot start with an underscore.

Reply:
Hi @petele , thanks. I just finished remove the underscore.